packages/viewer/src/components/viewer-transform-widget/viewer-transform-widget.spec.tsx

```diff
@@ -258,6 +258,117 @@ describe('vertex-viewer-transform-widget', () => {
     expect(onInteractionStarted).toHaveBeenCalled();
   });
 
+  it.only('performs a transform when initialized with a position', async () => {
```

Comment:
Is setting this test to be `.only` intentional here?

Reply:
Nope! Removed